src/locales/en/translation.json

```diff
@@ -424,19 +447,24 @@
     },
     "placeholders": {
       "redeem": "Redeem",
+      "useTicket": "Use my raffle ticket",
       "redeemNow": "Redeem Now",
       "viewItem": "View my tickets",
       "addTicket": "Add Ticket",
-      "addNewTicket": "Add New Ticket",
-      "addNewTickets": "Add New Tickets",
+      "addNewReceipt": "Add New Receipt",
+      "addNewReceipts": "Add New Receipts",
       "close": "Close",
       "activeRewardsLink": "View active rewards & giveaways",
       "markUsed": "Mark as used",
       "returnTo": "Return to passport",
       "prizeRedeemed": "PRIZE REDEEMED {{date}}",
-      "readyToRedeem": "READY TO REDEEM",
-      "leftToRedeem": "{{amount}} more stamps until your next reward",
-      "oneLeftToRedeem": "1 more stamp until your next reward"
+      "readyToRedeem": "Thank you for supporting small businesses",
+      "leftToRedeem": "{{amount}} more receipts to your next goal",
+      "oneLeftToRedeem": "1 more receipt to your next goal",
```

Comment:
i18n has a plural formatter that we can use instead of having to manually switching to the cases ourselves:
https://github.com/sendchinatownlove/sendchinatownlove.github.io/pull/444

Reply:
i didnt realize lol ty